src/components/domain/project/projectSearch/ProjectSearchBar.tsx

```diff
@@ -0,0 +1,76 @@
+'use client';
+
+import { useState } from 'react';
+import { IconInput } from '@/components/common/input/IconInput';
+import { Tabs, TabsList, TabsTrigger, TabsContent } from '@/components/ui/tabs';
+import { Search, User, Clipboard, ChevronDown, ChevronUp } from 'lucide-react';
+import { cn } from '@/lib/utils';
+
+interface ProjectSearchBarProps {
+  className?: string;
+}
+
+export default function ProjectSearchBar({ className }: ProjectSearchBarProps) {
+  const [placeholder, setPlaceholder] = useState('이름 혹은 이메일을 검색해주세요');
+  const [isDetailVisible, setIsDetailVisible] = useState(false);
```

Comment:
```suggestion
  const [isDetailVisible, toggleDetailVisibility] = useReducer((state) => !state, false);
```
토글만을 위한 state는 위와 같이 관리하는거로 통일시켰으면 합니다

Reply:
아 이 코드 익숙하지 않은 hook이라 그런지 자꾸 놓치네요. 수정하겠습니다!